awx/ui_next/src/util/useDeleteDetails.js

```diff
@@ -0,0 +1,23 @@
+export default async function getDeleteDetails(requests, count) {
+  const results = {};
+  let error = null;
+
+  if (count === 1) {
+    try {
+      const calls = await Promise.all(
+        Object.values(requests).map(request => request())
+      );
+      Object.keys(requests).forEach((key, index) => {
+        results[key] = calls[index].data.count;
+      });
+    } catch (err) {
+      error = err;
+    }
+  }
+  const hasCount = Object.values(results).some(value => value > 0);
```

Comment:
I think `count` and `hasCount` variables are a bit confusing because they refer to different counts. Maybe rename `count` to `numItemsToDelete` or something a little more specific